### DISCUSSION TITLE
Author-badges without author-partials

### DISCUSSION THREAD

**Discussion by JVimes** [nunocoracao/blowfish#955] [Ideas]
I can show authors. Or I can show authors+badges. But I found no way to show badges without authors, which would save page space with author links.

The author partials have author taxonomy links, just like badges. If it's redundant to show both, I suggest deprecating `showAuthorsBadges` in favor of something like:

```yml
multiAuthorAppearance = "none" # Options: none, full, badges
```

If that's not a good approach, I would suggest `showMultiAuthors` true/false. I avoid calling it "showAuthors" which is easily confused with showAuthor.

What do you think? 🙂

**Comment by wolfspyre** [comment]
I implemented this, and other flexibility as a POC and outlined it here: 
https://github.com/nunocoracao/blowfish/discussions/936

as I agree with you, the implementation, as-is feels (to me) a bit clunky.

would you envision this being a site-wide knob? 
tunable per section?
tunable per page?
tunable per individual? 

how granular would the control need to be to accommodate your use case?

**Reply by JVimes** [reply]
I think I'd do it like many existing configurations, where the default `multiAuthorAppearance` is set in params.toml, but it can be overridden per page in the frontmatter.
